src/AI-EditDistances/AIBrewDistance.class.st

```diff
@@ -0,0 +1,78 @@
+"
+Given two strings, obrain the number of edits required to make the first the same as the second. It use used typically when weighing multiple potentially similar strings in a data set.
+
+This algorithm separate costs for deletion (DEL) and insertion (INS) operations, so it allows you to differentiate between the two operations and assign different weights to them if needed. The edit cost can be configured by sending the `cost:` message to an instance of this class. The cost is an `Array` of positive `Integer` interpreted as:
+
+  - Match (0 is the default cost)
+  - Insertion (1 is the default cost)
+  - Deletion (1 is the default cost)
+  - Substitution (1 is the default cost)
+
+
+"
+Class {
+	#name : #AIBrewDistance,
+	#superclass : #AIAbstractEditDistance,
+	#instVars : [
+		'cost'
+	],
+	#category : #'AI-EditDistances-Distances'
+}
+
+{ #category : #accessing }
+AIBrewDistance >> cost [
+	"Answer an <Array> with the cost of edit operations as: 
+		1st element: Match
+		2nd element: Insert
+		3rd element: Deletion
+		4th element: Substitution "
+
+	^ cost
+		ifNil: [	cost := #( 0 1 1 1 ) ]
```

Comment:
Could we initialize the cost instance variable in the `initialize` method? something like:

``ìnitialize
    cost := self class defaultCostValue.
```
Because if we use this lazy initialization we are forced to use the accessor all the time.